content/docs/ui/account-and-settings/verifying-your-account.md

```diff
@@ -11,42 +11,43 @@ navigation:
 
-Complete one or more of the following:
+<call-out>
+
+If you don't have access to your domain's DNS records, you can use the **Send To A Coworker** feature. This feature allows you to enter a colleague's email address so Twilio SendGrid can send them all the information they need to complete domain authentication on your behalf. 
 
-* Verify your email address
-* Set up [two-factor authentication]({{root_url}}/ui/account-and-settings/two-factor-authentication/#setting-up-two-factor-authentication)
-* Set up [domain authentication]({{root_url}}/ui/account-and-settings/how-to-set-up-domain-authentication/)
+![]({{root_url}}/img/<domain-auth-send-to-coworker>.png "Send to a coworker tab")
 
-You may need to complete one or more of these steps to completely verify your account. You cannot skip any of the verification steps, or your account won't be completely unlocked.
+</call-out>
 
-### Resend the verification email
+2. Set up [two-factor authentication]({{root_url}}/ui/account-and-settings/two-factor-authentication/#setting-up-two-factor-authentication). Two-factor authentication allows you to make your SendGrid account more secure by requiring authentication beyond a simple username and password when logging in.
+3. Add a [payment method]({{root_url}}/ui/account-and-settings/billing/#payment-methods). Your card will not be charged until your first bill is due. 
 
-You can resend the verification to the email address you specified during the initial signup form by heading to the [verification guide](https://app.sendgrid.com/guide) and clicking "Resend in the "Confirm your Email Address" step. If you entered your email address incorrectly in the initial signup form, you can correct the email address and resend the verification email to the correct address.
+You will need to complete all of these steps to completely verify your account. You cannot skip any of the verification steps, or your account won't be unlocked. 
 
-## Setting up Two-factor Authentication
+<call-out type="warning">
 
-Two-factor authentication allows you to make your SendGrid account more secure by requiring authentication beyond a simple username and password when logging in. Learn more about setting up and using [two-factor authentication]({{root_url}}/ui/account-and-settings/two-factor-authentication/#setting-up-two-factor-authentication).
+In some instances our compliance team will need to manually verify your account information before you can start sending. In these cases your account should be unlocked within 72 hours of completing the last account verification step. 
 
-## Setting up Domain Authentication
+</call-out>
 
-Domain authentication shows email providers that SendGrid has your permission to send emails on your behalf. For more information, see [How to set up domain authentication]({{root_url}}/ui/account-and-settings/how-to-set-up-domain-authentication/).
 
 ## Upgrading your account
```

Comment:
I think you can remove this section now bc the previous section of this page covers it.